src/parameterizations/vertical/MOM_diabatic_aux.F90

```diff
@@ -1047,7 +1047,8 @@ subroutine applyBoundaryFluxesInOut(CS, G, GV, US, dt, fluxes, optics, nsw, h, t
               RivermixConst = -0.5*(CS%rivermix_depth*dt) * GV%Rho0 * ( US%L_to_Z**2*GV%g_Earth )
             endif
             cTKE(i,j,k) = cTKE(i,j,k) + max(0.0, RivermixConst*dSV_dS(i,j,1) * &
-                            (fluxes%lrunoff(i,j) + fluxes%frunoff(i,j)) * tv%S(i,j,1))
+                            (fluxes%lrunoff(i,j) + fluxes%frunoff(i,j) + &
```

Comment:
Please add parentheses to specify the order of the sum of these 4 runoff terms.

Reply:
Addressed in 69220e9cebf8bfdade695c9e617fd6002d99c84a.